styles/main.css

```diff
@@ -0,0 +1,364 @@
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 0.2fr 1fr;
+   grid-template-areas: "video video aside"
+       "section section aside";    
+}
+
+section{
+    grid-area: section;
+    display: grid;
+    grid-gap: 1em;
+    grid-template-rows: auto 1fr;
+}
+
+.section__comments {
+    width: 26.4%;
+    position: relative;
+    top: 1em;
+}
+
+.section__sort-container {
+    position: relative;
+    left: 10em;
+    bottom: 0.3em;
+    width: 20%;
+}
+
+.section__sort-container p {
+    position: relative;
+    left: 2.5em;
+    bottom: 1.5em;
+    color: gray;
+    font-size: 0.9em;
+}
+
+.section__sort-container a {
+    text-decoration: none;
+}
+
+.section__profile {
+    width: 8%;
+}
+
+.section__profile img {
+    width: 3.3em;
+}
+
+.input__container {
+    position: relative;
+    left: 5em;
+    bottom: 3em;
+    margin-right: 5em;
+}
+
+.input__container input {
+    width: 100%;
+    border: none;
+    border-bottom: 1px solid lightgray;
+    color: gray;
+    font-size: 1em;
+}
+
+.comments p,
+.comments a {
+    margin-left: 4.5em;
+    position: relative;
+}
+
+.comments__name {
+    position: relative;
+    bottom: 3em;
+    font-size: 0.86em;
+    font-weight: 700;
+}
+
+.comments__name a{
+    text-decoration: none;
+    color: gray;
+    position: relative;
+    right: 4em;
+    font-weight: 700;
+}
+
+.comments__comment {
+    position: relative;
+    bottom: 2em;
+    line-height: 1.3em;
+}
+
+.likes {
+    display: grid;
+    grid-template-columns: repeat(4, 3em);
+    width: 20em;
+    font-size: 0.86em;
+    position: relative;
+    bottom: 2em;
+}
+
+.comments__text {
+    position: relative;
+    top: 0.5em;
+    text-decoration: none;
+    color: gray;
+}
+
+.comments__replies {
+    text-decoration: none;
+    color: #4885ed;
+    position: relative;
+    bottom: 0.9em;
+}
+
+.main__video {
+    grid-area: video;
+    border-bottom: 1px solid lightgray;
+    padding-bottom: 2em;
+}
+
+.main__video,
+section {
+    font-size: 0.9em;
+}
+
+.main__description p:first-of-type {
+    margin-bottom: 1em;
+}
+
+.main__description p:last-of-type {
+    float: left;
+}
+
+.main__description img {
+    float: right;
+}
+
+.main__description {
+    border-bottom: 1px solid lightgray;
+    margin-bottom: 1em;
+    padding-bottom: 2em;
+    margin-top: 1em;
+}
+
+.main__name {
+    float: left;
+}
+
+.main__subscriptions {
+    float: right;
+}
+
+.main__sub-description p {
+    margin-top: 2em;
+    float: left;
+    line-height: 2em;
+}
+
+.main__sub-description p:first-of-type {
+    margin-right: 6em;
+}
+
+.main__more {
+    margin-top: 14em;
+    font-size: 0.85em;
+}
+
+.main__more a {
+    text-decoration: none;
+    color: gray;
+}
+
+.main__video-container {
+  position: relative;
+  padding-bottom: 52.4%; /*16:9*/
+  padding-top: 30px; 
+  height: 0; 
+  overflow: hidden;
+}
+ 
+.main__video-container iframe {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
```

Comment:
Be consistent with the indentation, some times you use 1 space and other you use more than one.